### PR TITLE
Set up application structure

### PR DESCRIPTION
**Pull request by ghost**
### What we have learned

- How to scaffold a new rails application

### What we did

- Scaffolded new rails application
- Cleaned up unnecessary comments for good readability

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+gem 'rails', '~> 5.2.0'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (tochman) [inline Gemfile]: Please remove coffee

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
```
Comment (tochman) [inline Gemfile]: Please remove jbuilder

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (tochman) [inline Gemfile]: Please set up pry the way we showed you in the original walkthrough

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: Please remove this line

File: README.md [hunk 5]
```diff
@@ -1,2 +1,3 @@
 # Food Hub
 April 2018 CA Cohort Mid Course Project
+Set up application structure
```
Comment (tochman) [inline README.md]: Why is this added to the README?

File: config/application.rb [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    config.generators do |generate|
+      generate.helper false
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+      generate.system_tests false
+    config.generators.system_tests = nil
```
Comment (tochman) [inline config/application.rb]: IF you choose to keep this line (it is redundant) please place it outside the current do — end block.

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <%= yield %>
+  </body>
+</html>
```
Comment (tochman) [inline app/views/layouts/application.html.erb:15]: Please add a chore to PT to set up HAML as template engine

**Review by tochman — CHANGES_REQUESTED** [review]
Small changes needed before merge

**Comment by tochman** [comment]
I need you to work in pairs and have the commit history reflect that.

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: Tested to see if PR was set up properly. It has been removed.

File: config/application.rb (hunk 6 above)
Comment (ghost) [reply config/application.rb]: Please confirm which line you are referring to, is 'config.generators.system_test =nil' ?

File: app/views/layouts/application.html.erb (hunk 7 above)
Comment (ghost) [reply app/views/layouts/application.html.erb:15]: We added a chore to PT for HAML,
PT link https://www.pivotaltracker.com/story/show/157386405

File: config/application.rb (hunk 6 above)
Comment (tochman) [reply config/application.rb]: Line 27. The one right above the comment. That’s the standard

**Review by tochman — APPROVED** [review]
